config/config.properties

```diff
@@ -448,6 +448,15 @@ extendeds3.configUri=http://localhost:9020?identity=user&secretKey=password
 # Prefix is optional.
 # extendeds3.prefix=
 
+# Size of ECS objects in bytes above which it is no longer considered a small object.
+# This value is used to optimize transactions performance when size of transaction segments is small.
+# For small transaction segments, to implement concat ExtendedS3Storage reads complete source segment and appends it to target
+# instead of using multipart upload.
+# smallObjectSizeLimitForConcat is optional.
+# Valid values: Positive integer.
```

Comment:
Please call out what is the default value, which really matters in the case.
Also consider saying something like: "In most cases, the default value should just work. Change this value with cautiousness."

Reply:
Good point. Updated